runtime/near-vm-logic/src/logic.rs

```diff
@@ -396,9 +387,8 @@ impl<'a> VMLogic<'a> {
     /// * It's up to the caller to set correct len
     #[cfg(feature = "sandbox")]
     fn sandbox_get_utf8_string(&mut self, len: u64, ptr: u64) -> Result<String> {
-        self.try_fit_mem(ptr, len)?;
         let mut buf = vec![0; len as usize];
```

Comment:
This is sandbox-only so not a huge deal if we don’t actually prevent people shooting themselves in the foot, but it probably wouldn't hurt anybody if we validated `len` here regardless.